build/Targets/Analyzers.Imports.targets

```diff
@@ -180,14 +192,6 @@
 
   <!-- ====================================================================================
        
-         Generation of AssemblyVersion attributes from the BuildVersion property
-       
-       ==================================================================================== -->
-
-  <Import Project="GenerateAssemblyInfo.targets" Condition="'$(ProjectLanguage)' == 'CSharp' OR '$(ProjectLanguage)' == 'VB'" />
```

Comment:
This is now done using the auto-generation of assembly attributes in the Microsoft.NET.Sdk.